### DISCUSSION TITLE
How to add attendee?

### DISCUSSION THREAD

**Discussion by kobazzo** [spatie/laravel-google-calendar#190] [Q&A]
I have done all suggested for configure the laravel project and calendar synch with service account.
It's work if i don't add an attendee to the event, otherwise i have this error:
"Error 403 - Service accounts cannot invite attendees without Domain-Wide Delegation of Authority"

How can i add attendees?

**Comment by fuhrn** [comment]
https://github.com/spatie/laravel-google-calendar/discussions/210#discussioncomment-789871
